sysutils/sfm/Portfile

```diff
@@ -8,11 +8,12 @@ PortGroup               legacysupport 1.0
 github.setup            afify sfm 0.4 v
 categories              sysutils
 license                 ISC
-maintainers             {@sikmir gmail.com:sikmir} openmaintainer
-platforms               darwin
+maintainers             {@sikmir disroot.org:sikmir} openmaintainer
 description             Simple file manager
 long_description        ${name} is a simple file manager for unix-like systems.
 
 checksums               rmd160  cbe737b636406924d5b11a1f200ba133b75165ab \
                         sha256  2446bcd8c27903aaaac128e08993308ea107023f822eedb058a855c5469c96ae \
                         size    29519
+
```

Comment:
I think a comment here explaining why this is in the file might be useful, as it's pretty unusual.